.clippy.toml

```diff
@@ -1,4 +1 @@
 allow-mixed-uninlined-format-args = false
-disallowed-types = [
-    { path = "tower::util::BoxCloneService", reason = "Use our internal BoxCloneService which is Sync" },
```

Comment:
I think we should still disallow it, but in favor of the other tower type.

Reply:
done :)